main.tf

```diff
@@ -10,7 +10,7 @@ terraform {
 // Configure providers
 // ----------------------------------------------------------------------------
 provider "aws" {
-  version = ">= 2.28.1"
+  version = ">= 2.53.0"
```

Comment:
I am not sure whether this is a good idea. I used to be also in the camp of trying to use the latest and even being more restrictive in the versions allowed (using the ~> operator), but if you look at this issue https://github.com/jenkins-x/terraform-google-jx/issues/84 I am not so sure anymore whether this is a good idea. I think @cagiti changed my mind on this. If you read the referenced Terraform documentation:

>Shared modules should constrain only the minimum allowed version, using a >= constraint. This specifies the minimum version the provider is compatible with while allowing users to upgrade to newer provider versions without altering the module source code.

I understand this as use '>= 2.28.1' until you are forced to use another minimal version.

Reply:
So, the reason I upgraded the version to 2.53.0 (the latest is 2.65.0) is because I bumped the version for the vpc module to 2.39.0 (https://github.com/terraform-aws-modules/terraform-aws-vpc/blob/master/versions.tf), I guess I could use ~> instead of >=.

Reply:
I think that's fine. If we are saying that _2.53.0_ is needed in order to upgrade the vpc module which in turn resolves some of the issues we have, then I am all in. 
I just wanted to make sure that we are not updating for the sake of it and that my understanding with the Terraform provider versions is that one should be as liberal as possible.